presto-cli/src/main/java/io/prestosql/cli/TerminalUtils.java

```diff
@@ -19,15 +19,20 @@
 import java.io.IOException;
 import java.io.UncheckedIOException;
 import java.nio.charset.Charset;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import static com.google.common.base.Preconditions.checkState;
 
 public class TerminalUtils
 {
     private static final Terminal TERMINAL_INSTANCE = createTerminal();
+    private static final AtomicBoolean isOpened = new AtomicBoolean(true);
```

Comment:
The `is` prefix is normally only used for boolean getter methods, like the `get` prefix for regular getters.

It seems better to call this `closed`, since that's the typical name for this type of field.